schema.graphql

```diff
@@ -169,6 +169,28 @@ type FpmmParticipation @entity {
   timeout: BigInt
 }
 
+enum TradeType {
+  Buy
+  Sell
+}
+
+type FpmmTrade @entity {
+  id: ID!
+  fpmm: FixedProductMarketMaker!
+
+  # lifted from fpmm
+  title: String
+  collateralToken: Bytes!
+
+  type: TradeType!
+  creator: String!
+  creationTimestamp: BigInt!
+  amount: BigInt!
+  feeAmount: BigInt!
+  outcomeIndex: BigInt!
+  outcomeTokensTrade: BigInt!
```

Comment:
`outcomeTokensTraded` corresponds better with `outcomeTokensBought` and `outcomeTokensSold`.

Reply:
In the README of this repo, the only name that appears is `outcomeTokensBought`, however when I try to query a trade requesting that field I get an error 
`"message": "Cannot query field \"outcomeTokensBought\" on type \"FpmmTrade\". Did you mean \"outcomeTokensTraded\"?",`

Someone renamed the README but the schema is the same as the old one. Could someone then explain to me what is supposed to be outcomeTokensTrade? Is the amount of tokens bought?
Thanks